tests/automated-benchmarking.py

```diff
@@ -0,0 +1,189 @@
+import argparse
+import glob
+import os
+import re
+
+import numpy as np
+
+
+GITHUB_URL = "https://github.com/facebook/zstd"
+WORKING_DIR = "zstd-automated-benchmarking"
+LEVELS = [1, 2, 3, 4, 5]
+BUILDS = [
+    {"compiler": "clang", "mode": "64"},
+    {"compiler": "gcc", "mode": "64"}
+]
+
+# Not sure what the threshold for triggering alarms should be
+# 1% regression sounds like a little too sensitive but the desktop
+# that I'm running it on is pretty stable so I think this is fine
+CSPEED_REGRESSION_TOLERANCE = 0.01
```

Comment:
Try to make it possible to modify these values on command line.
You can keep constants in the source code as "default" ones, it's a good practice.

This makes the script a lot more flexible, eventually usable on different platforms than current target one.
Generally speaking, consider that any script in `/tests` can be used by any user or contributor of `zstd`.
This elusive "user" will eventually happen to be you in a few months or years, and you'll be glad that the script is flexible enough and do not make too many hard-coded assumptions.

Reply:
Yep!